src/lib/terkin/driver/hx711_sensor.py

```diff
@@ -110,22 +118,52 @@ def read(self):
         if self.loadcell is None:
             return self.SENSOR_NOT_INITIALIZED
 
+        address = self.address
+
         #log.info('Acquire reading from HX711')
-        reading = self.loadcell.get_reading()
+        readingA = self.loadcell.get_reading()
+
+        # with dualchannel 3 data sets are generated instead of one
+        # 1 - cumulated weight, no parameter, 2 - channel A reading with parameters, 3 - same for channel B
+        if self.parameter['dualchannel']:   # read channel B
```

Comment:
Please say `self.parameter.get('dualchannel')` here to improve backwards-compatibility.